sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -27,12 +27,12 @@ internal FieldValue(FieldValue_internal fieldValue, IReadOnlyList<ReadResult> re
         /// Initializes a new instance of the <see cref="FieldValue"/> structure.
         /// </summary>
         /// <param name="value">The actual field value.</param>
-        /// <param name="isPhoneNumber">Whether or not this value represents a phone number.</param>
-        internal FieldValue(string value, bool isPhoneNumber = false)
+        /// <param name="type">The data type of the field value.</param>
+        internal FieldValue(string value, FieldValueType type)
             : this()
         {
-            ValueType = isPhoneNumber ? FieldValueType.PhoneNumber : FieldValueType.String;
             ValueString = value;
+            ValueType = type;
```

Comment:
I still think we should check that the type is either string, phonenumber, or countrycode. otherwise the user could pass any type and this will be confusing